### PR TITLE
Feature/mobile version works section

### PR DESCRIPTION
**Pull request by Beardless-sheik**
-Update index.html to add "works" section
-Update style.css to add styling to added markup
-Added background images based on the template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,15 +60,90 @@ <h3 class="headline-text-1">Alick Nyirenda</h3>
...
         <li><a href="#"><i class="social-icon fa fa-angellist clr-secondary"></i></a></li>
         <li><a href="#"><i class="social-icon fa fa-twitter"></i></a></li>
         <li><a href="#"><i class="social-icon fa fa-medium"></i></a></li>
     </ul>
+    <div class="work-arrow-container">
+      <span class="work-arrow"></span>
+      <p class="work-arrow-text">Work</p>
+    </div>
+  </section>
+
```
Comment (Temmarie) [inline index.html]: - [ ] You have a horizontal overflow. Kindly fix it.
- [ ] The social media icons are a lot smaller than the ones in the design template. Kindly fix it.
_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  image | image

File: index.html [hunk 2]
```diff
@@ -60,15 +60,90 @@ <h3 class="headline-text-1">Alick Nyirenda</h3>
...
+        <ul class="work-card-skills-container">
+          <li class="work-card-skills-description">CSS</li>
+          <li class="work-card-skills-description">Bootstrap</li>
+          <li class="work-card-skills-description">Ruby</li>
+          <li class="work-card-skills-description">Express</li>
+        </ul>
+        <div class="see-project-button">
+          <a href="#"> <p class="see-project-text">See Project</p> </a>
+        </div>
+      </div>
```
Comment (Temmarie) [inline index.html]: - [ ] I see that you've used `ul`-`li` to order the project technologies, Good job! However, the project itself is a list and should be ordered using `ul`-`li` as well. 
Please try to use `ul` and `li` for semantic correctness whenever you are trying to list items. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.


- [ ] Adjust the `see project` button to have the same style as the design template.

_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @Beardless-sheik     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Recommended** ♻️
_See the inline comments below for the required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: index.html (hunk 2 above)
Comment (Beardless-sheik) [reply index.html]: Made changes in the 2 newest commits

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html]: Well resolved the issue!

File: index.html (hunk 2 above)
Comment (DeliceLydia) [reply index.html]: - [ ] Please work on the previous change requested by the reviewer the projects itself is a list and each project has a list inside it.. let me give you an example below
<ul>projects
     <li>1st project
       <ul>
          <li>css</li>
          <li>bootstrap</li>
       </ul>
      </li>
   </ul>
   
- [ ] I can see that you tried to work on the button but still you can improve compare your page as the design

ds
Try to stick to the design as much as you can.. I suggest that you center the project section to make it look good

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Beardless-sheik, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments I added under the previous review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Beardless-sheik) [reply index.html]: Quick question also: Do the projects themselves also need to be an ordered list. For example: should the .h4 tag, img placeholder tag and paragraph tag be in a list too??

Comment (henatan99) [reply index.html]: Good job on this one!!! :clap: 
- [ ] OPTIONAL: The dividing-line `div` should not be in the list though. 
- [ ] OPTIONAL: You may also need to work on alignment. The contents should be aligned centered as in the original design. 
Screenshot from 2021-06-12 19-57-01

Comment (henatan99) [reply index.html]: - [ ] Button looks better now but when hover over text color should not be changed. You may fix this in your spare time. 
Screenshot from 2021-06-12 20-00-15

**Review by henatan99 — APPROVED** [review]
Hi @Beardless-sheik , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- You may want to see OPTIONAL comments I wrote you under previews reviews comments. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**